web/graph.template.tail

```diff
@@ -34,30 +50,59 @@
 	  </center>
 	</td>
 	<td style="vertical-align:top">
-          COLLDESC collections:
-	  <ul>
-	  COLLECTIONS
-	  </ul>
-	  <p>
-	  ACTIVESHARDS
-	  <p>
-	  MAINTSHARDS
-	  <p>
-          <a href="LEADERBOARDURL">Top 25 contributors</a>:<br>
-	  <i>(<a href="#contribute">you can help!</a>)</i>
-          <ol>
+    <div class="panel panel-primary">
+      <div class="panel-heading">COLLDESC collections:</div>
+      <div class="panel-body">
+	    <ul>
+	    COLLECTIONS
+	    </ul>
+    </div>
+  </div>
+  <div class="panel panel-primary">
+    <div class="panel-heading">SHARDS</div>
+    <div class="panel-body">
+      <div class="panel panel-info">
+        <div class="panel-heading">Active</div>
+        <div class="panel-body">
+          ACTIVESHARDS
```

Comment:
Javascript sounds like the least-invasive way to fix it. We could certainly have a second template for shards, or conditionals, or whatever.

If we gave the panel an id, then it would be a simple matter of hiding it when document.querySelector("#theid .panel-body").childNodes.length == 0.